tests/test_helpers.py

```diff
@@ -1,21 +1,31 @@
-from stanley.helpers import get_receiver, get_sender
+import pytest
+
+from stanley.helpers import get_receiver, get_sender, get_filtered_member
 from stanley.redis import redis_storage
 from stanley.settings import REDIS_KEY_RECEIVE_FEEDBACK, REDIS_KEY_SEND_FEEDBACK
 
 
-def test_get_sender(members):
+@pytest.fixture
+def feedback_members_env():
```

Comment:
You are adding complexity with a fixture that is used once. Why not just patching the value in place?